compiler/rustc_middle/src/mir/interpret/mod.rs

```diff
@@ -355,6 +364,16 @@ impl<'s> AllocDecodingSession<'s> {
                     let alloc_id = decoder.interner().create_fn_alloc(instance);
                     alloc_id
                 }
+                AllocDiscriminant::Vtable => {
+                    assert!(alloc_id.is_none());
+                    trace!("creating static alloc ID");
+                    let ty = <Ty<'_> as Decodable<D>>::decode(decoder);
+                    let poly_trait_ref =
+                        <Option<ty::PolyExistentialTraitRef<'_>> as Decodable<D>>::decode(decoder);
+                    trace!("decoded vtable alloc instance: {ty:?}, {poly_trait_ref:?}");
+                    let alloc_id = decoder.interner().create_vtable_alloc(ty, poly_trait_ref);
```

Comment:
Does this make any sense? I was pretty much guessing here.

Reply:
yea, this is in sync with the existing scheme, which has worked well so far